cwms-data-api/src/main/java/cwms/cda/data/dao/JooqDao.java

```diff
@@ -128,6 +130,16 @@ public static DSLContext getDslContext(Context ctx) {
         return retVal;
     }
 
+    protected static Timestamp buildTimestamp(Instant date)
+    {
+        Timestamp retval = null;
+        if(date != null)
+        {
```

Comment:
The curlies on this are wrong.

Also this would probably read easier just doing a `return date !=null ? Timestamp.from(date) : null;`

Reply:
Updated to fix brackets and used a ternary